tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py

```diff
@@ -213,10 +212,11 @@ def test_resource_size(self):
             },
         )
 
-        data = response.data["data"]
         assert response.status_code == 200
+
+        data = response.data["data"]
         assert len(data) == 2
-        assert data[0][1][0]["count"] == 0.0
+        assert not data[0][1][0]["count"]
```

Comment:
In the metrics layer test, the data[0][1][0]["count"] is `None` while the non-metrics layer it is `0.0`. 

I found this a bit odd, but i'm assuming this might be some implementation difference between using the metrics layer and not using it, so i'm checking if its Falsey to get the test passing.

Reply:
Yeah, this is unfortunately a divergence in how the filling of missing timeseries ranges is done on both ends (one does zerofilling the other just `None`). What would be your expected behavior?

Reply:
I do think None makes the most sense, this way the client can determine what to do if there is missing data.